PFR_solver_CANTERA.m

```diff
@@ -0,0 +1,43 @@
+function [F] = PFR_solver_CANTERA(x,initial,gas,mdot,A_in,dAdx,k)

```

Comment:
Instead of naming this function `PFR_solver_CANTERA`, can you just name it `PFR_solver`? I think you should also be able to include it in the `PFR_setup.m` file, at the end of that file. I'm not that familiar with how MATLAB functions are defined though.

Reply:
yes.